specification/communication/data-plane/TrunkStatus/preview/2022-07-01-preview1/communicationservicestrunkstatus.json

```diff
@@ -0,0 +1,327 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Trunk health Service",
```

Comment:
This service seems fairly minimal in scope, why didn't we integrate it with the SipRouting service? The overhead of another service and SDK isn't small and I would assume as a developer I can't use TrunkHealth if I don't also use SipRounting?

Reply:
Could the health data inlined with the rest of the Trunk properties in the SipRouting service?